### DISCUSSION TITLE
Alignment under ZBox

### DISCUSSION THREAD

**Discussion by Solido** [btwld/mix#436] [Q&A]
Hi!

First I'm glad to see Mix going forward, I started using it at the beginning but now it's really getting difficult to not use it.

Trying to push it forward, can you help me with the right style to position under a Zbox aka Positioned or Align values.

Here are my tests with results:

```dart
ZBox(
  children: [
    Box(
      style: Style(
        $stack.alignment(Alignment(1, 1)), // KO Box stay top left of the stack
        $box.alignment(Alignment(.5, 1)), // KO align inside Box only
        $box.alignment.bottomCenter(), // OK at the stack bottom 
        // To to use explicit aligment on the ZBox itself?
      )
  ]
)

```

Thank you!

**Comment by leoafarias** [comment]
@Solido You can use widget modifiers `$box.wrap.align()`. This will wrap the `Box` widget with an `Align` widget. Does that make sense?

**Reply by Solido** [reply]
Yes! I did not catch this ref until I've read the scale effect. 
But I still miss `Positioned`. 

The framework is very potent but translation from Flutter itself require more **code exemples**
so we can just copy paste the basics and grasp the feature beyond the documentation.

I've had to revert to Flutter when working with ZBox.
Hope to convert asap.

Thanks!
